debug/dev-seafile-logs.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd /var/lib/seafile/logs/
```

Comment:
why `cd`ing into a directory when you can `tail -f /var/lib/seafile/logs/seafile.log`?